presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriverUri.java

```diff
@@ -215,7 +225,8 @@ public void setupClient(OkHttpClient.Builder builder)
                 builder.addInterceptor(basicAuth(getUser(), password));
             }
 
-            if (useSecureConnection) {
+            SslVerificationMode sslVerificationMode = SSL_VERIFICATION.getValueOrDefault(properties).orElse(FULL);
```

Comment:
@kokosing I am not sure how to avoid calling `orElse(FULL)` after `getValueOrDefault(properties)`.
May I ask for your advice? Thank you.

Reply:
Replace it with `orElseThrow()`. We expect it to have a default value, so it should fail otherwise.

Reply:
Resolved. Thank you.